packages/ses/scripts/bundle.js

```diff
@@ -15,10 +17,17 @@ const write = async (target, content) => {
 };
 
 const main = async () => {
-  const bundle = await makeBundle(
+  const bytes = await fs.promises.readFile(
+    fileURLToPath(`${root}/package.json`),
+  );
+  const text = textDecoder.decode(bytes);
```

Comment:
Can we not do this:

```suggestion
  const test = await fs.promises.readFile(
    fileURLToPath(`${root}/package.json`),
    'utf8'
  );
```

and drop the `TextDecoder`?  Is this an Endo convention?

Reply:
nice the 2nd arg w `utf8` cleans up the need for `TextDecoder` ^
spotted `utf-8` is also used, but don't think it matters? (same result)
if not, then RE convention i favour our suggestion (the default)

<img width="1164" alt="Screenshot 2023-11-20 at 4 39 04 pm" src="https://github.com/endojs/endo/assets/1881059/f8b056b8-801f-497d-8435-e7743d3a89b2">